src/main/java/frc/robot/subsystems/Chamber.java

```diff
@@ -136,20 +138,37 @@ public boolean shootNote(double leftSpeed, double rightSpeed) {
     }
   }
 
+  public boolean isNoteAligned() {
```

Comment:
This shouldn't be a public method, because all we care about from outside this class is whether a note is chambered. That implies alignment.

Reply:
This is not resolved.